cdm/src/main/java/ucar/nc2/ft2/coverage/simpgeometry/SimpleGeometryCoverage.java

```diff
@@ -125,46 +190,71 @@ public void toString(Formatter f, Indent indent) {
   public SimpleGeometryCS getCoordSys() {
     return coordSys;
   }
-
-  ///////////////////////////////////////////////////////////////
-
-  // LOOK must conform to whatever grid.readData() returns
-  // LOOK need to deal with runtime(time), runtime(runtime, time)
-  public List<CoordinateAxis> getAxis() {
-        return coordSys.getSimpleGeometryID();
+  
+  /**
+   * Retrieves the x Axis that corresponds to this geometry.
+   * 
+   * @return x axis
+   */
+  public CoordinateAxis getXAxis() {
+	return xAxis;  
+  }
+  
+  /**
+   * Retrieves the y Axis that corresponds to this geometry.
+   * 
+   * @return y axis
+   */
+  public CoordinateAxis getYAxis() {
+	return yAxis;  
+  }
+  
+  /**
+   * Retrieves the z Axis that corresponds to this geometry.
+   * 
+   * @return z axis
+   */
+  public CoordinateAxis getZAxis() {
+	return zAxis;  
+  }
+  
+  /**
+   * Retrieves the ID Axis that corresponds to this geometry.
+   * 
+   * @return id axis
+   */
+  public CoordinateAxis getIDAxis() {
+	return IDAxis;  
   }
 
   @Override
   public boolean hasMissing() {
-    return true;
+    return false;

```

Comment:
Why has the return value been changed?

Reply:
Good catch! Unfortunately that function isn't actually implemented yet- will go do that shortly (or remove the interface that uses that entirely).